backend/data/src/main/kotlin/io/tolgee/repository/NotificationRepository.kt

```diff
@@ -0,0 +1,30 @@
+package io.tolgee.repository
+
+import io.tolgee.model.Notification
+import org.springframework.context.annotation.Lazy
+import org.springframework.data.domain.Page
+import org.springframework.data.domain.Pageable
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Query
+import org.springframework.stereotype.Repository
+
+@Repository
+@Lazy
+interface NotificationRepository : JpaRepository<Notification, Long> {
+  @Query(
+    """
+    SELECT n
+     FROM Notification n
+     LEFT JOIN FETCH n.project
```

Comment:
Because we have the enhancers, you probably don't want to fetch it, right?

Reply:
True, removed.